src/Framework/Sdk/SdkResultItem.cs

```diff
@@ -16,6 +21,7 @@ public class SdkResultItem
 
         public SdkResultItem()
         {
+            ItemSpec = "";
```

Comment:
Hmm...I hadn't noticed this, but it implies that ItemSpec is supposed to be able to be null—in theory, at least. With the #nullable check, that would prevent the second constructor from letting you pass in a null itemspec, right? I don't think anyone is doing that, but theoretically a breaking change? I don't think it's actually something we should really worry about, but tiny nit:
```suggestion
            ItemSpec = string.Empty;
```

doesn't allocate a new string.

Reply:
Thanks! apply this now.